app/core/config.py

```diff
@@ -101,10 +101,9 @@ class Settings(BaseSettings):
     LOG_DEBUG_MESSAGES: bool | None
     # Hyperion follows Semantic Versioning
     # https://semver.org/
-    HYPERION_VERSION: str = "1.0.0"  # This value should never be modified by hand. See [Hyperion release] documentation
-    MINIMAL_TITAN_VERSION_CODE: int = 100
-    # Depreciated, minimal_titan_version_code should be used
-    MINIMAL_TITAN_VERSION: str = "0.0.1"
+    HYPERION_VERSION: str = "0.0.1"
+    MINIMAL_TITAN_VERSION_CODE: int = 1
+    MINIMAL_TITAN_VERSION: str = "0.0.1"  # deprecated, use MINIMAL_TITAN_VERSION_CODE
```

Comment:
Why should we give a default value to these three version numbers ?

I think we want to throw an error if they are not defined in the .env

Reply:
In my opinion these fields should be hardcoded and not based on the dotenv. The `MINIMAL_TITAN_VERSION_CODE` is known during development and not deployement.
Allowing to override the values in the dotenv may sometimes be usefull.

Reply:
I am not sure to understand what you mean by
known during development and not deployement

The issue I see with leaving a default value is that we might release a Hyperion without adding the versions and noticing it since it will not produce any errors

Reply:
I am resolving this comment as the real issue seems to be that we are moving theses variable to .env